packages/aws-cdk-lib/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -82,6 +142,18 @@ export class Accelerator extends cdk.Resource implements IAccelerator {
    */
   public readonly dnsName: string;
 
+  /**
+   * IP addressess associated with the accelerator.
+   */
+  public readonly ipAddresses?: string[];
```

Comment:
Depending on [this comment](https://github.com/aws/aws-cdk/pull/28055/files#r1400194833), it will be removed.